resources/monitoring/values.yaml

```diff
@@ -1408,7 +1408,7 @@ prometheusOperator:
   resources:
     limits:
       cpu: 300m
-      memory: 500Mi
+      memory: 1Gi
```

Comment:
If you do the big increase here, you can remove it from production profile

Reply:
Makes sense, will do